frontend/src/metabase/css/core/base.styled.ts

```diff
@@ -10,7 +10,7 @@ export const rootStyle = css`
   height: 100%; /* ensure the entire page will fill the window */
   display: flex;
   flex-direction: column;
-  background-color: var(--mb-color-bg-light);
+  background-color: var(--mb-color-bg-primary);
```

Comment:
Note to self - I think this actually needs to be `bg-secondary` and then some other overrides aren't needed.